ml-kem/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
```

Comment:
It'd be good to add a toplevel Cargo.toml with a `[workspace]` configuration. [Here's an example](https://github.com/RustCrypto/sponges/blob/master/Cargo.toml)